DynamicActiveRecord.php

```diff
@@ -44,22 +44,43 @@
  */
 class DynamicActiveRecord extends ActiveRecord
 {
-    /**
-     * Prefix for base64 encoded dynamic attribute values
-     */
-    const DATA_URI_PREFIX = 'data:application/octet-stream;base64,';
+    private $_dynamicAttributes = [];
+
+    protected static $_encoder;
```

Comment:
i generally only use `$_` for privates. since this is a cache for a getter, shouldn't it be private anyway?